apps/ghost-blog-buster/src/commands/interactive-admin-api/prompt-credentials-loop.ts

```diff
@@ -46,12 +46,12 @@ export const promptCredentialsLoop = async (config: Configstore) => {
       process.exit(0);
     }
     try {
-      const ghost = new TSGhostAdminAPI(`${url.protocol}//${url.hostname}`, ghostAdminApiKey, "v5.0");
+      const ghost = new TSGhostAdminAPI(url.origin, ghostAdminApiKey, "v5.0");
       s.start("Validating credentials");
       const res = await ghost.site.fetch();
       if (res.success) {
         const settings = res.data;
-        config.set("ghostUrl", `${url.protocol}//${url.hostname}`);
+        config.set("ghostUrl", url.origin);
```

Comment:
I didn't notice earlier you were also storing the url in order to reuse it later.
I've applied the change in those places too.